src/Integration.Vsix.UnitTests/Analysis/AnalysisStatusNotifierTests.cs

```diff
@@ -56,7 +58,8 @@ public void AnalysisStarted_LogToOutputWindow()
             testSubject.AnalysisStarted();
 
             var expectedMessage = string.Format(AnalysisStrings.MSG_AnalysisStarted, filePath);
-            logger.AssertOutputStringExists(expectedMessage);
+            logger.AssertPartialOutputStringExists(AnalyzerName);
```

Comment:
The link between the individual tests and the expected `analyzerName` is a bit indirect.
It would be better if the analyzer name was passed to `CreateTestSubject`.